neomodel/match.py

```diff
@@ -7,13 +7,24 @@
 
 def rel_helper(**rel):
     if rel['direction'] == OUTGOING:
-        stmt = '-[{0}:{1}]->'
+        stmt = '-{0}->'
     elif rel['direction'] == INCOMING:
-        stmt = '<-[{0}:{1}]-'
+        stmt = '<-{0}-'
     else:
-        stmt = '-[{0}:{1}]-'
-    ident = rel['ident'] if 'ident' in rel else ''
-    stmt = stmt.format(ident, rel['relation_type'])
+        stmt = '-{0}-'
```

Comment:
I suppose direction should be explicit, defaulting to ```(x)--(y)``` may have unexpected results for users, if I understand your question correctly.